src/widget/wtrackwidgetgroup.cpp

```diff
@@ -0,0 +1,125 @@
+
+#include "widget/wtrackwidgetgroup.h"
+
+#include <QDebug>
+#include <QStylePainter>
+#include <QUrl>
+
+#include "control/controlobject.h"
+#include "util/dnd.h"
+#include "widget/wtrackmenu.h"
+
+namespace {
+
+int kDefaultTrackColorAlpha = 255;
+
+const WTrackMenu::Features kTrackMenuFeatures =
+        WTrackMenu::Feature::Playlist |
+        WTrackMenu::Feature::Crate |
+        WTrackMenu::Feature::Metadata |
+        WTrackMenu::Feature::Reset |
+        WTrackMenu::Feature::BPM |
+        WTrackMenu::Feature::Color |
+        WTrackMenu::Feature::FileBrowser |
+        WTrackMenu::Feature::Properties;
+
+} // anonymous namespace
+
+WTrackWidgetGroup::WTrackWidgetGroup(QWidget* pParent,
+        UserSettingsPointer pConfig,
+        TrackCollectionManager* pTrackCollectionManager,
+        const QString& group)
+        : WWidgetGroup(pParent),
+          m_group(group),
+          m_pConfig(pConfig),
+          m_trackColorAlpha(kDefaultTrackColorAlpha),
+          m_pTrackMenu(make_parented<WTrackMenu>(
+                  this, pConfig, pTrackCollectionManager, kTrackMenuFeatures)) {
+    setAcceptDrops(true);
+}
+
+WTrackWidgetGroup::~WTrackWidgetGroup() {
```

Comment:
Please replace with `= default`, either in the .h or the .cpp file depending on the dependencies.